src/components/operations/operation-details/ko/runtime/operation-details.ts

```diff
@@ -244,6 +256,19 @@ export class OperationDetails {
         this.working(false);
     }
 
+    public async loadExamples(operation: Operation): Promise<void> {
+        const representations = operation.request.meaningfulRepresentations().map(a => {return {...a}});
+        let requestExamples = {};
+        if (representations && representations.length) {
+            for(var i = 0; i < representations.length; i++) {
+                let value = representations[i].examples?.[0];
+                if (!value) return;
+                requestExamples[representations[i].contentType] =  ko.observable(value.title);
+            }
+        }
+        this.selectedRepresentatnionsValue(requestExamples);
```

Comment:
```suggestion
        const requestExamples = operation.request.meaningfulRepresentations()
        .map(a => {return {...a}})
        .filter(a => a.examples?.[0])
        .reduce( (a, c) => {
          c[a.contentType] = ko.observable(a.title);
          return c;
        }, {});
        this.selectedRepresentatnionsValue(requestExamples);
```

Reply:
This is only a suggestion to use build in functions to accomplish same result  :) No need to apply.

Reply:
Thank you for the suggestion. 
May I clarify if this is mandatory change? 
In terms of the performance is there any difference?

Reply:
I is not a mandatory change.

I do not think there is a major difference in performance